src/commands/optimize.ts

```diff
@@ -81,6 +82,10 @@ export default class Optimize extends Command {
     this.outputMethod = flags.output as Outputs;
     this.metricsMetadata.optimized = false;
 
+    if (flags['disableOptimizationForSchema'] === true) {
+      flags.disableOptimizationForSchema = true;
+    }
```

Comment:
You basically wrote
```ts
if (flags['disableOptimizationForSchema'] === true) {
  flags['disableOptimizationForSchema'] = true;
}
```
cuz just to make sure.

Reply:
I tried set the flag so it can be used in the optimizedDocument.

Reply:
The very presence of a command line switch in this CLI already means 'true' and its absence - 'false', so you already imported the value of this cmd switch into the JS execution context in lines `R106-R108`.
It's done in `Bundler` the same way:
https://github.com/asyncapi/cli/blob/master/src/commands/bundle.ts#L44

This makes code in lines `R85-R87` redudant and it can be deleted.

Reply:
Oh yes, my bad. I don't know how I missed it. Thank you. I will incorporate all the necessary changes in the future commit to this PR.